src/vmm/src/persist.rs

```diff
@@ -32,10 +34,9 @@ use seccompiler::BpfThreadMap;
 use snapshot::Snapshot;
 use versionize::{VersionMap, Versionize, VersionizeResult};
 use versionize_derive::Versionize;
+use virtio_gen::virtio_ring::VIRTIO_RING_F_EVENT_IDX;
 use vm_memory::GuestMemoryMmap;
 
-#[cfg(target_arch = "x86_64")]
-const FC_V0_23_SNAP_VERSION: u16 = 1;
 #[cfg(target_arch = "x86_64")]
 const FC_V0_23_MAX_DEVICES: u32 = 11;
```

Comment:
nit: should this also go in `version_map.rs` const definitions?

Reply:
I'm not sure. Since the check is implemented in this file I would keep it here. But if you have strong preferences I can move it.